web/typescript/page/edit.ts

```diff
@@ -0,0 +1,13 @@
+export function editLimit(selectorQery, maxLen) {
```

Comment:
Is it necessary to accomplish this with Javascript? I'd prefer a HTML-only solution for this.

Reply:
The `pattern` attribute never worked for textareas.

Reply:
For the record, my suggestion:

```ts
export function checkMaxLength() {
    [...document.querySelectorAll('textarea[maxlength]')].forEach((element) => {
        const maxlength = element.getAttribute('maxlength');
        element.setAttribute('maxlength', '');
        element.addEventListener('input', () => {
             element.setCustomValidity(maxlength >= element.value.length ? 'toolong' : '');
        }, {passive: true});
    });
}
```